templates/kubernetes/terraform/modules/kubernetes/variables.tf

```diff
@@ -144,6 +144,24 @@ variable "user_auth" {
   }))
 }
 
+variable "enabled_dev_env_user_auth" {
+  description = "When enabled will provision Kratos and Oathkeeper Rules for dev environment"
+  type        = bool
+  default     = false
+}
+
+variable "dev_user_auth_frontend_domain" {
+  description = "Frontend domain used for local development with dev env"
+  type        = string
+  default     = "127.0.0.1:3000"
+}
+
+variable "dev_user_auth_frontend_scheme" {
```

Comment:
AFAICT this scheme and domain are only ever concatenated, never used independently. Is there a reason to keep them separate in the vars?

Reply:
just checked, its always used together 
hmm I guess we should merge them, the only downside would be is its a breaking change